docs/_automation/waste-reduction-logic-app/README.md

```diff
@@ -0,0 +1,71 @@
+
+# Automated detection
+
+The **Waste Reduction Logic App** is an automated detection mechanism provided by [Azure Logic App](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-overview). The logic app will run on a configurable schedule to monitor selected subscriptions for a set of idle resources and send notifications once it finds any of those resources to alert admins to investigate and take action.
+
+Use the following steps to deploy the Logic App
+
+1. Create a new **Custom Deployment**
+    > [!IMPORTANT]
+    > The logic app needs to be in the same region as its resource group
+   * Select one of the following options to deploy the [main.bicep](https://github.com/microsoft/finops-toolkit/blob/f81e0669ddf104e3db1d5fe21b470f0231e4d936/src/logic-apps/waste-reduction/main.bicep).
+     * [Cloud Shell](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/deploy-cloud-shell?tabs=azure-cli)
+     * [PowerShell](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/deploy-powershell)
+     * [VS Code](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/deploy-vscode)
+     * [CLI](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/deploy-cli)
+   * Verify the deployment is successful
+
+2. Authorize Connection
+    > [!NOTE]
+    > After the deployment is complete, the API connection will have an error. This is expected and will be fixed after authorizing the connection
+
+   * Select the **API connection** resource, then select **Edit API Connection** in the General tab to authorize the connection
+    ![Edit API connection page](images/Edit-API-connection.png)
+   * Select **Authorize**
+
+    <br>
+  
+    > [!IMPORTANT]
+    > The account authorizing the connection will be used by the Logic App to send the Alerts
+
+   * Select **Save** after the authorization is successful
+    ![Authorize & Save page](images/Authorize-&-Save.png)
+   * Go back to the **Overview** blade and verify that the **Status: Connected**
+
+3. Create a system-assigned identity to allow the Logic App to "read" the resources in the subscription
```

Comment:
Can we make this part of the template? I suppose we can do that in a future release, if needed, but it'd be unfortunate to have the extra manual steps.